src/test/java/com/bwell/domain/ResourceLoaderTest.java

```diff
@@ -79,14 +79,4 @@ public void testLoadingBundleContainedResources() throws JSONException, IOExcept
         String patient_first_identifier = identifier1.getValue();
         assertEquals("12345", patient_first_identifier);
     }
-
-    @Test
-    public void testLoadingBundleWithBadFhirDiv() throws JSONException, IOException {
```

Comment:
why was this unit test method removed?

Reply:
I removed that test, because it’s not failing. Not sure why that JSON would fail to parse, but it no longer fails to parse in the new code.

Good question. There was no new tests added. 

I could add a test with the contained issue and then assert the issue doesn't exist. 

I will do that and then commit to this PR before we merge.

Reply:
This has now been corrected. Test uploaded